appeals/web/src/server/lib/mappers/lpaQuestionnaire.mapper.js

```diff
@@ -29,6 +32,34 @@ export function initialiseAndMapLPAQData(
 	session,
 	currentRoute
 ) {
+	/**
+	 * Generates a document field with defaults
+	 *
+	 * @param {Object} options
+	 * @param {string} options.id
+	 * @param {string} options.text
+	 * @param {FolderInfo|null|undefined} options.folderInfo
+	 * @param {string} [options.cypressDataName]
+	 * @returns {Instructions}
+	 */
+	const documentField = ({ id, text, folderInfo, cypressDataName }) => {
```

Comment:
As mentioned elsewhere I think this should be more descriptively named. I'm not sure why the wrapper function is necessary if all it's doing is calling `documentTypeDisplayField` and returning the result. Also there are variables here being referenced from the containing scope - I think those should be passed in as parameters for clarity

Reply:
Those parameters are the same for all uses in the mapper, so the idea is to reduce ~4 lines of copied code. With this approach it is clear those parameters are the same for each row.

Reply:
I guess clarity is subjective, personally I think this approach is sacrificing clarity for brevity, which is a trend I generally disagree with in principle - it's trivial in this case, but in my experience, a lot of small deviations from the established convention which individually seem trivial can add up to a significant drop in clarity overall. But it's not a hill I'm willing to die on in this case